2020-10-28 20:38:59.486 - [Info] MITM Version: 1.26
2020-10-28 20:38:59.492 - [Info] Auto Access Enabled: true
2020-10-28 20:38:59.493 - [Debug] [Init] Auto Access Barrier: true
2020-10-28 20:38:59.516 - [Info] SSH man-in-the-middle server for 172.20.0.202 listening on 0.0.0.0:10001
2020-10-28 20:39:54.036 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-libssh_0.11
2020-10-28 20:39:55.200 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-28 20:39:55.201 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-28 20:39:55.470 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-28 20:39:55.473 - [Debug] [Auto Access] Attacker: 195.3.147.47, Threshold: 1, Attempts: 1
2020-10-28 20:39:55.473 - [Debug] [Auto Access] Compromising the honeypot
2020-10-28 20:39:55.473 - [Debug] [Auto Access] Adding the following credentials: "admin:admin"
2020-10-28 20:39:56.206 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-28 20:39:56.207 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.202
2020-10-28 20:39:56.311 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-28 20:39:58.130 - [Debug] [Connection] Attacker closed connection
2020-10-28 20:39:58.135 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-28 20:39:58.136 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-28 20:40:16.376 - [Debug] [Connection] Attacker connected: 65.49.20.69 | Client Identification: SSH-2.0-Go
2020-10-28 20:40:16.824 - [Debug] [Auth] Attacker 65.49.20.69 trying to authenticate with "none"
2020-10-28 20:40:16.825 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-28 20:40:21.362 - [Debug] [Connection] Attacker closed the connection
2020-10-28 20:40:35.027 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-PuTTY_Release_0.61
2020-10-28 20:40:36.144 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-28 20:40:36.144 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-28 20:40:36.404 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-28 20:40:36.405 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-28 20:40:36.406 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-28 20:40:38.225 - [Debug] [Connection] Attacker closed the connection
2020-10-28 20:43:38.629 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-PuTTY_Release_0.64
2020-10-28 20:43:40.112 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-28 20:43:40.112 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-28 20:43:40.373 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-28 20:43:40.374 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-28 20:43:40.376 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-28 20:43:42.224 - [Debug] [Connection] Attacker closed the connection
